front/tests/002-project-management.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { test, expect } from '@playwright/test';
+import { v4 as uuidv4 } from 'uuid';
+import { ProjectResult } from 'common/api/osrdEditoastApi';
+import { PlaywrightHomePage } from './pages/home-page-model';
+import project from './assets/operationStudies/project.json';
+import { ProjectPage } from './pages/project-page-model';
+import PlaywrightCommonPage from './pages/common-page-model';
+import { deleteApiRequest, getApiRequest, postApiRequest } from './assets/utils';
+
+let newProjectData: ProjectResult;
+
+test.beforeEach(async () => {
+  newProjectData = await postApiRequest('/projects/', {
```

Comment:
This code is also in study-management  scenario-management. Maybe we can find a way to make it reusable?

Reply:
it should be possible, but those beforeEach and afterEach interact with an external variable newProjectData wich is used in every tests in association with an uuid(), setting those globally may turn the code more complicated than we need